sub-app/src/singleSpa.js

```diff
@@ -10,13 +10,7 @@ const reactLifecycles = singleSpaReact({
 });
 
 export const parcelConfig = {
-    bootstrap() {
-        return reactLifecycles.bootstrap;
```

Comment:
You actually have to call the bootstrap() function. You can't just return it.

Reply:
thx! seems like a copy-paste error